dashboard/app/views/layouts/application.html.haml

```diff
@@ -6,6 +6,7 @@
     = render partial: 'i18n/crowdin_in_context_tool'
     = render inline: File.read(Rails.root.join('..', 'shared', 'haml', 'onetrust_cookie_scripts.haml')), type: :haml, locals: {dashboard: true, domain: 'code.org'}
     - appname = Rails.env.production? ? t(:appname) : "#{t(:appname)} [#{Rails.env}]"
+    - managed_test_server = "#{CDO.running_web_application? && CDO.test_system?}"
```

Comment:
did you ever figure out why this was set wrong? (can totally be done in a separate PR if not)

Reply:
I don't think I was injecting ruby in the correct way before so am abstracting it out into its own variable to match what else is happening in this file!